src/components/Router/CrossDeviceMobileRouter.tsx

```diff
@@ -99,7 +99,9 @@ export default class CrossDeviceMobileRouter extends Component<
     // the cross device flow without running nginx
     const url = props.urls.sync_url
 
-    const roomId = window.location.pathname.substring(3) || props.options.roomId
+    // TODO: review this change
+
+    const roomId = props.options.roomId || window.location.pathname.substring(3)
```

Comment:
When making the crossdevice working locally, it was always taking the input of the url in preference to the roomId passed as options. The or seems to be flipped. 

Please review @ethan605

Reply:
As discussed, it should be safe to change this as we only need to fetch the room id from the URL when the roomId is not set in the store. So if roomId is null or undefined we can get it from the URL